src/helpers/worker-hours-info.model.ts

```diff
@@ -49,7 +77,7 @@ export class WorkerHourInfo {
   constructor(workerHourNorm: number, workerTime: number, overTime: number) {
     this.workerHourNorm = Math.round(workerHourNorm);
     this.workerTime = Math.round(workerTime);
-    this.overTime = overTime;
+    this.overTime = Math.round(overTime);
```

Comment:
Tylko dlatego, żeby nie mieszać bo we wszystkich innych miejscach jest `round`.

Reply:
A jednak wróciło XD